scripts/firmware_log_parser.py

```diff
@@ -0,0 +1,513 @@
+#!/usr/bin/env python3
+"""
+This script parse firmware log
```

Comment:
Please add a short readme

Reply:
inside a new folder called `fw_log_parser` next to the parser

Reply:
May I add the `HWLoggerEventsDS5.xml` file to the folder because I wrote about him in a readme?

Reply:
No need, it's from the FW team